app/models/dispatcher.rb

```diff
@@ -0,0 +1,5 @@
+class Dispatcher < ApplicationRecord
+  validates :name, presence: true
+  validates :email, presence: true, uniqueness: true
+  validates :password, presence: true
+end
```

Comment:
@DmitrS please add a newline to the end of the file :construction_worker_man:

You can config Sublime to do it automatically for you: https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842/2